### PR TITLE
PDPD Place: remaining methods

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
 - *Implementation of remaining methods for PDPD Place class*
 - *Unit tests*

### Tickets:
 - *52252*

### CODE REVIEW

File: ngraph/test/frontend/frontend_manager.cpp [hunk 1]
```diff
@@ -141,12 +141,16 @@ TEST(FrontEndManagerTest, testDefaultPlace)
     ASSERT_ANY_THROW(place->get_names());
     ASSERT_ANY_THROW(place->get_consuming_operations());
     ASSERT_ANY_THROW(place->get_consuming_operations(0));
+    ASSERT_ANY_THROW(place->get_consuming_operations("", 0));
```
Comment (nosovmik) [inline ngraph/test/frontend/frontend_manager.cpp:144]: also add `get_consuming_operations("")` to achieve 100% code coverage :-)

File: ngraph/frontend/paddlepaddle/src/place.cpp [hunk 2]
```diff
@@ -25,17 +25,187 @@ bool PlacePDPD::is_output() const
...
+                consuming_ops.end(), consuming_ops_out.begin(), consuming_ops_out.end());
+        }
+    }
+    return consuming_ops;
+}
+
+std::vector<Place::Ptr> OpPlacePDPD::get_consuming_operations(const std::string& outputPortName,
+                                                              int outputPortIndex) const
+{
+    return get_output_port(outputPortName, outputPortIndex)->get_consuming_operations();
```
Comment (nosovmik) [inline ngraph/frontend/paddlepaddle/src/place.cpp:131]: Implementation looks quite generic. If it is going to be the same for other frontends - can we consider to move this to base class implementation? Same is for other similar methods

File: ngraph/test/frontend/frontend_manager.cpp (hunk 1 above)
Comment (itikhono) [reply ngraph/test/frontend/frontend_manager.cpp:144]: done in https://github.com/openvinotoolkit/openvino/pull/6637

File: ngraph/frontend/paddlepaddle/src/place.cpp [hunk 3]
```diff
@@ -25,17 +25,187 @@ bool PlacePDPD::is_output() const
...
+                              const std::string& name)
+{
+    m_input_ports[name].push_back(input);
+}
+
+Place::Ptr OpPlacePDPD::get_output_port(const std::string& name) const
+{
+    FRONT_END_GENERAL_CHECK(
+        m_output_ports.at(name).size() == 1,
+        "There are multiple OutputPort Places with the same name, please specify index.");
```
[the remark this answers is not in the document]
Comment (itikhono) [reply ngraph/frontend/paddlepaddle/src/place.cpp:86]: fixed in openvinotoolkit#6637

File: ngraph/frontend/paddlepaddle/src/place.cpp [hunk 4]
```diff
@@ -25,17 +25,187 @@ bool PlacePDPD::is_output() const
...
+        m_input_ports.at(name).size() == 1,
+        "There are multiple InputPort Places with the same name, please specify index.");
+    return m_input_ports.at(name)[0];
+}
+
+Place::Ptr OpPlacePDPD::get_output_port() const
+{
+    FRONT_END_GENERAL_CHECK(m_output_ports.size() == 1 &&
+                                m_output_ports.begin()->second.size() == 1,
+                            "There are multiple OutputPort Places, please specify index.");
```
[the remark this answers is not in the document]
Comment (itikhono) [reply ngraph/frontend/paddlepaddle/src/place.cpp:102]: fixed in openvinotoolkit#6637

File: ngraph/frontend/paddlepaddle/src/place.cpp [hunk 5]
```diff
@@ -25,17 +25,187 @@ bool PlacePDPD::is_output() const
...
+        m_output_ports.at(name).size() == 1,
+        "There are multiple OutputPort Places with the same name, please specify index.");
+    return m_output_ports.at(name)[0];
+}
+
+Place::Ptr OpPlacePDPD::get_input_port(const std::string& name) const
+{
+    FRONT_END_GENERAL_CHECK(
+        m_input_ports.at(name).size() == 1,
+        "There are multiple InputPort Places with the same name, please specify index.");
```
[the remark this answers is not in the document]
Comment (itikhono) [reply ngraph/frontend/paddlepaddle/src/place.cpp:94]: fixed in openvinotoolkit#6637

**Pull request closed without merging**